package.json

```diff
@@ -56,45 +57,55 @@
   },
   "dependencies": {
     "angular": "npm:@xlts.dev/angular@1.9.1",
-    "core-js": "^3.20.3"
+    "angular-animate": "npm:@xlts.dev/angular-animate@1.9.1",
+    "angular-aria": "npm:@xlts.dev/angular-aria@1.9.1",
+    "angular-material": "npm:@xlts.dev/angular-material@^1.2.4",
```

Comment:
I know that such a comment was added to the [tracking document](https://docs.google.com/document/u/2/d/1Ncayh2Au9x5Uz377l4gVEC4jk_NSd3_4ZVKw6PNpBMY/edit). However, the changes I made were done before this document was written. I wanted the repository consistent with` angularjs-angular-hybrid-angular-cli`, but it only applied to `AngularJS `without `TypeScript`. The added code improved the project configuration, for example, adding `parserOptions `in the `eslint `configuration.

Reply:
I do have a preference for keeping things simple with these demo apps (i.e. not include extra dependencies unless they are necessary or we want to demonstrate something), but I don't feel too strongly about it.

I'll defer to @Splaktar, because we had discussed that we don't want to introduce a new dependency.
If this is fine with him, it's fine with me.

Reply:
I agree with @gkalpak, and I specifically spoke with @jzolnowski and let him know that we should not add AngularJS Material to this repo.

That said since the work is already done, I will look at it and decide if we should keep it. I will need to run the app locally to do that, so I won't be able to take care of that right now.

Reply:
Based on the lack of functionality in the original version of this app and the fact that this work is already done, I think it is acceptable to add this additional dependency.